src/tequila/quantumchemistry/qc_base.py

```diff
@@ -1638,7 +1638,8 @@ def rdm2(self):
             return None
 
     def compute_rdms(self, U: QCircuit = None, variables: Variables = None, spin_free: bool = True,
-                     get_rdm1: bool = True, get_rdm2: bool = True, ordering="dirac", use_hcb: bool = False):
+                     get_rdm1: bool = True, get_rdm2: bool = True, ordering="dirac", use_hcb: bool = False,
+                     rdm_trafo: QubitHamiltonian = False):
```

Comment:
default should be `None` instead of `False`
otherwise `if rdm_trafo is None` further down doesn't work as intended